config/checkstyle/checkstyle.xml

```diff
@@ -341,46 +340,62 @@
     <!--
     JAVADOC CHECKS
     -->
+    <!-- Checks that all block-tags are ordered correctly. -->
+    <module name="AtclauseOrder"/>
+
+    <!-- Checks that Javadoc block tags appear only at the beginning of the line. -->
+    <module name="JavadocBlockTagLocation"/>
+
+    <!-- Checks that all Javadoc comments start from the second line. -->
+    <module name="JavadocContentLocationCheck">
+      <property name="location" value="second_line"/>
```

Comment:
I think we can remove this line, from [the documentation](https://checkstyle.sourceforge.io/config_javadoc.html#JavadocContentLocation) the default value is `second_line`.